tests/gtests/test_eltwise.cpp

```diff
@@ -68,9 +68,25 @@ class eltwise_test : public ::testing::TestWithParam<eltwise_test_params> {
         ref_dst.reset(new memory({*data_desc, *eng}));
 
         data_t data_median = data_t(0);
-        data_t data_deviation
-                = p.alg_kind == eltwise_elu || p.alg_kind == eltwise_exp || p.alg_kind == eltwise_swish
-                ? data_t(1) : data_t(200);
```

Comment:
Instead of a switch: `? data_t(1) : p.alg_kind == eltwise_hswish ? data_t(6) : data_t(200);`